packages/components/input/src/input.js

```diff
@@ -0,0 +1,55 @@
+import { TSElement, unsafeCSS, html, customElementDefineHelper } from '@tradeshift/elements';
+import css from './input.css';
+
+customElementDefineHelper(
+	'ts-input',
+	class extends TSElement {
+		static get styles() {
+			return [TSElement.styles, unsafeCSS(css)];
+		}
+
+		static get properties() {
+			return {
+				disabled: { type: Boolean, attribute: 'disabled', reflect: true }
+			};
+		}
+
+		attributeChangedCallback(name, oldVal, newVal) {
+			const input = this.getSlotInput();
+			if (name === 'disabled' && input) {
+				input.disabled = newVal !== undefined && newVal !== null;
+			}
```

Comment:
What's the reason for this?
Why do we need the disabled state inside the component, if it's only for styling

Reply:
I was thinking to support disabled status both on the ts-input and the input in the slot. 
May be we need to support disabled form that all the elements in the form should be disabled. 
This if or sync the disable status from the host to slot.